Cargo.toml

```diff
@@ -30,6 +30,11 @@ version = "0.21"
 features = ["download-libtorch"]
 optional = true
 
+[dependencies]
+async-trait = "0.1.83"
+reqwest = { version = "0.11", features = ["json"] }
+tokio = { version = "1.0", features = ["full"] }
```

Comment:
Shouldn't the rt and macros features suffice?

Reply:
Also, should be a dev dependency?

Reply:
Hey @danwritecode.
Where is Tokio used outside of tests?

Reply:
you're right, just updated it.